Unit3/view/frontend/templates/customtab.phtml

```diff
@@ -0,0 +1,23 @@
+<?php
+
+$helper = $this->helper('RLTSquare\Unit3\Helper\Data\Data');
+
+if ($helper->isEnable()) : ?>
+
+<strong>Username:</strong>
+<?php echo $this->helper('RLTSquare\Unit3\Helper\Data\Data')->getConfig('unit3/general/username'); ?><br>
+<strong>Password:</strong>
+<?php
+$type = $this->helper('RLTSquare\Unit3\Helper\Data\Data')->getConfig('unit3/general/dropdown');
+$pass = $this->helper('RLTSquare\Unit3\Helper\Data\Data')->getConfig('unit3/general/password');
+if ($type == 'Staging' || $type == 'Development') {
```

Comment:
Not good ... instead of 'Development' and 'Stagging', you could have declared constant in your options class, You should declare value keys here as constants, e.g. "const ENV_STAGING = 'staging'", so you can later use RLTSquare\UnitThree\Model\Config\Source\Select::ENV_STAGING in if statements and other comparisons you need.